### DISCUSSION TITLE
Print_Summary Output Error

### DISCUSSION THREAD

**Discussion by bi-glitch** [CamDavidsonPilon/lifelines#1462] [Q&A] [answered]
![troubleshoot](https://user-images.githubusercontent.com/80623801/191957278-d455bfce-30a5-46bf-92a8-1baef1bb4b7c.png)

Whenever I try to use print_summary from any of the lifelines statistical outputs, I receive this error with no statistical information and an empty dataframe. I am using the program in a Jupyter notebook in DataSpell, is there a way for me to resolve this?

**Comment by CamDavidsonPilon** [comment, marked as answer]
Hm, you can try: `results.print_summary(style="ascii")` and let me know if that works?

**Reply by bi-glitch** [reply]
Yes, that worked, thank you!

**Reply by CamDavidsonPilon** [reply]
Looks like a bug in `to_latex` - will be fixed in the next version of lifelines
